web/rootfs/defaults/meet.conf

```diff
@@ -37,6 +39,22 @@ location = /http-bind {
     proxy_set_header Host {{ .Env.XMPP_DOMAIN }};
 }
 
+{{ if $ENABLE_XMPP_WEBSOCKET }}
+# xmpp websockets
+location = /xmpp-websocket {
+    proxy_pass {{ .Env.XMPP_BOSH_URL_BASE }}/xmpp-websocket;
+    proxy_http_version 1.1;
+
+    proxy_set_header Connection "upgrade";
+    proxy_set_header Upgrade $http_upgrade;
+
+    proxy_set_header Host {{ .Env.XMPP_DOMAIN }};
+    proxy_set_header X-Forwarded-For $remote_addr;
+    proxy_read_timeout 60s;
```

Comment:
> A value higher than the socket framework network_settings.read_timeout setting is advisable. (default 840 seconds in trunk).
(https://prosody.im/doc/websocket#nginx)
Any thoughts on this?

Reply:
We haven't play with this much yet, but I think it's fine. We want to get a timeout on inactivity and this one is specifically for the `/xmpp-websocket` path.